### PR TITLE
global job-reconfiguring

### PR DESCRIPTION
**Pull request by naumvd95**

### CODE REVIEW

File: servers/partner-ci/9.0/contrail.yaml [hunk 1]
```diff
@@ -26,9 +26,10 @@
             VSRX_ORIGINAL_IMAGE_PATH=/storage/contrail/vSRX.400.img;
             VSRX_TARGET_IMAGE_PATH=/var/lib/libvirt/images/vSRX.400.img;
             VSRX_TEMPLATE_PATH=/storage/contrail/vSRX_400.template;
+            FORCE_DISABLE_UPDATES=true;
 - project:
     name: 'predefined_parameters'
-    fuel-release: '9.1'
+    fuel-release: '9.0'
```
Comment (rkhozinov) [inline servers/partner-ci/9.0/contrail.yaml:32]: it seems that it can break job_naming

Comment (rkhozinov) [reply servers/partner-ci/9.0/contrail.yaml:32]: may be we should think about a special variable in COMMON_PROPS

Comment (naumvd95) [reply servers/partner-ci/9.0/contrail.yaml:32]: it would be a separate job, so renaming -good case

**Comment by naumvd95** [comment]
please specify branches repo to each job

File: servers/partner-ci/9.0/contrail.yaml (hunk 1 above)
Comment (rkhozinov) [reply servers/partner-ci/9.0/contrail.yaml:32]: it's ok. but we have to take into account that 9.0 version is used for 9.0 and 9.1

**Review by rkhozinov — APPROVED** [review]

**Pull request merged**